programs/ssl/ssl_client1.c

```diff
@@ -99,6 +102,13 @@ int main( void )
     /*
      * 0. Initialize the RNG and the session data
      */
+#if defined(MBEDTLS_PLATFORM_C)
+    if( ( ret = mbedtls_platform_setup( &platform_ctx ) ) != 0 )
+    {
+        mbedtls_printf( " failed\n  !  mbedtls_platform_setup returned %d\n\n", -ret );
```

Comment:
We're displaying a negative return value here, yet at line 239 are displaying it without the negative. We should be self-consistent within the same program.